uint16 index
float32 value
---
bool success